ibis/tests/expr/test_table.py

```diff
@@ -1357,14 +1357,12 @@ def test_multiple_dbcon():
     con1.table('alltypes').union(con2.table('alltypes')).execute()
 
 
-@pytest.mark.xfail(
-    raises=RelationError, reason='Expr from different DB should fail #64'
-)
 def test_multiple_db():
     con1 = MockBackend()
     con2 = MockAlchemyBackend()
 
-    con1.table('alltypes').union(con2.table('alltypes')).execute()
+    with pytest.raises(RelationError):
+        con1.table('alltypes').union(con2.table('alltypes')).execute()
```

Comment:
Can pull out as much of this as you can from the context manager so it's more clear what is raising `RelationError`?